packages/paste-core/components/time-picker/src/index.tsx

```diff
@@ -1,2 +1,3 @@
-export * from "./TimePicker";
-export * from "./utils";
+export { TimePicker } from "./TimePicker";
+export type { TimePickerProps } from "./TimePicker";
```

Comment:
suggestion: omit "element" from input props on TimePickerProps and add it explicitly in order to provide the default "TIMEPICKER"